interface/VideoPostProcessDefs.h

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
seems it not install in $install_dir/include/libyami. seems you need add it libyami_vpp_source_h